server/main.lua

```diff
@@ -51,7 +51,7 @@ local function respawn(src)
 
 		for hospitalName, hospital in pairs(Config.Locations.hospitals) do
 			if hospitalName ~= 'jail' then
-				if not closest or #(coords - hospital.coords) < closest then
+				if not closest or #(coords - hospital.coords) < #(coords - closest) then
```

Comment:
micro optimization to save closest distance as well to avoid re-computing. No action required.